perun-base/pom.xml

```diff
@@ -139,6 +139,13 @@
 			<version>${testcontainers.version}</version>
 			<scope>compile</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.datatype/jackson-datatype-jsr310 -->
+		<dependency>
+    		<groupId>com.fasterxml.jackson.datatype</groupId>
+    		<artifactId>jackson-datatype-jsr310</artifactId>
+    		<version>2.13.1</version>
```

Comment:
I think we can omit the version tag, because this project has `spring-boot-starter-parent` as a parent project. It should automatically provide versions for some dependencies and update them in the future.